doc/classes/ProjectSettings.xml

```diff
@@ -916,7 +916,8 @@
 			Changing this value allows setting up a multi-project scenario where there are multiple [code].csproj[/code]. Keep in mind that the Godot project is considered one of the C# projects in the workspace and it's root directory should contain the [code]project.godot[/code] and [code].csproj[/code] next to each other.
 		</member>
 		<member name="editor/export/convert_text_resources_to_binary" type="bool" setter="" getter="" default="true">
-			If [code]true[/code], text resources are converted to a binary format on export. This decreases file sizes and speeds up loading slightly.
+			If [code]true[/code], text resources are converted to a binary format on export, for example, it will turn [code]tscn[/code] to [code]scn[/code] and [code]tres[/code] to [code]res[/code]. This decreases file sizes and speeds up loading slightly. To access project resources once exported, it is recommended to use [ResourceLoader] instead of [FileAccess].
+			[b]Note:[/b] [ProjectSettings] is not a resource so [code]project.godot[/code] will always be converted to [code]project.binary[/code] on export.
```

Comment:
- No need to acknowledge the exported binary file's extensions (`scn`/`res`), since accessing them by raw file path on a release project is heavily discouraged.
- "_To access project resources once exported [...]_" is not related to the purpose of, and therefore fits snuggly in a note (although I'm not sure it's needed).
- "_ProjectSettings is not a resource so [...]_" is a weird justification to me. _Something_ happens to `project.godot` but it is unaffected by this setting.

```suggestion
			If [code]true[/code], text resource ([code]tres[/code]) and text scene ([code]tscn[/code]) files are converted to their corresponding binary format on export. This decreases file sizes and speeds up loading slightly.
			[b]Note:[/b] Because a resource's file extension may change in an exported project, it is heavily recommended to use [method @GDScript.load] or [ResourceLoader] instead of [FileAccess] to load resources dynamically.
			[b]Note:[/b] The project settings file ([code]project.godot[/code]) will always be converted to binary on export, regardless of this setting.
```

Reply:
Thanks!